contribs/gnodev/cmd/gnobro/main.go

```diff
@@ -326,8 +332,8 @@ func runServer(ctx context.Context, gnocl *gnoclient.Client, cfg *broCfg, bcfg b
 			bubbletea.Middleware(teaHandler),
 			activeterm.Middleware(), // ensure PTY
 			ValidatePathCommandMiddleware(bcfg.URLPrefix),
-			logging.StructuredMiddlewareWithLogger(
-				charmlogger, charmlog.DebugLevel,
+			StructuredMiddlewareWithLogger(
```

Comment:
future improvement: maybe a recover() middleware, so that the logger shows errors "gracefully"?